### PR TITLE
qcram: section reference format

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: draft-ietf-quic-qcram.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ the number of concurrently blocked streams.
 
 ### Fixed overhead.
 
-HPACK defines overhead as 32 bytes ({{!RFC7541}} Section 4.1).  As described
+HPACK defines overhead as 32 bytes ({{!RFC7541}}, Section 4.1).  As described
```
Comment (afrind) [inline draft-ietf-quic-qcram.md]: Is there a consistent format we should use for references to RFC7541?  Or it's ok that one is 'see Section X.Y of' and the other is ', Section X.Y'?

File: draft-ietf-quic-qcram.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ PUSH_PROMISE frames (see {{hq-frames}}).
 If a header block contains no vulnerable header fields, BLOCKING MUST be 0.
 This implies that the header fields are represented either as references
 to dynamic table entries which are known to have been received, or as
-Literal header fields (see {{RFC7541}} Section 6.2).
+Literal header fields (see Section 6.2 of {{RFC7541}} ).
```
Comment (afrind) [inline draft-ietf-quic-qcram.md]: Is there an extraneous space before ) ?

File: draft-ietf-quic-qcram.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-qcram.md]: I have been using "Section X.Y of {{CITATION}}" always.  The other acceptable form is "{{CITATION}}, Section X.Y", but there is value in consistency.

File: draft-ietf-quic-qcram.md (hunk 2 above)
Comment (reschke) [reply draft-ietf-quic-qcram.md]: yes

File: draft-ietf-quic-qcram.md (hunk 1 above)
Comment (reschke) [reply draft-ietf-quic-qcram.md]: ...depends on context. The first form works a bit better in prose, the second one in parentheses. IMHO.

**Pull request merged**